### DISCUSSION TITLE
Is there any way I can add global tags?

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#12] [Q&A]
Note: this discussion was created from a user reaching out to our support team.

Is there any way I can add global tags? One that I will reuse in new snippets. Currently snippets are created with either the "Flutter" or "flutter" tag and there is no consistency in character size. Additionally, in many cases I have to rewrite the same tags, which defeats the purpose. There should be an option to use the ones I added, and the ability to sort them. In addition, is there any way to sort snippets into categories like in notepad?

**Comment by rosie-at-pieces** [comment]
@mark-at-pieces any thoughts on this?

**Comment by peaklabs-dev** [comment]
I think all tags should be workspace-specific e.g. workspace global. So Flutter tag is available in every snippet in a workspace and is the same for all snippets so we can filter for tags, flutter small is also available in every snippet and is the same for the whole workspace, but tags are case-sensitive, so Flutter and flutter are two different Golbas tags. --> Also let us filter for tags
